.gitignore

```diff
@@ -77,8 +77,6 @@ local.properties
 *.log
 
 # Build results
-[Dd]ebug/
-[Rr]elease/
```

Comment:
It could. I thought those were hangovers from olden days so I just got rid. Shall I add them back?